packages/client/src/console-window.ts

```diff
@@ -23,13 +23,35 @@ export class ConsoleWindow implements Window {
         return Promise.resolve(undefined);
     }
 
-    createOutputChannel(name: string): vscode.OutputChannel {
+    createOutputChannel(name: string, _options: string | {
+        log: true;
+    } | undefined): vscode.LogOutputChannel {
         const existing = this.channels.get(name);
         if (existing) {
             return existing;
         }
-        const channel: vscode.OutputChannel = {
+
+        const channel: vscode.LogOutputChannel = {
             name: 'default',
+            logLevel: vscode.LogLevel.Info,
+            get onDidChangeLogLevel(): vscode.Event<vscode.LogLevel.Info> {
+                return {} as vscode.Event<vscode.LogLevel.Info>;
```

Comment:
Does it work? it's supposed to be a function, you can use `onDidChangeLogLevel = new vscode.EventEmitter<vscode.LogLevel.Info>().event`

Reply:
btw, all that ConsoleWindow stuff is optional and will probably disapear soon

Reply:
> Does it work? it's supposed to be a function, 

Thanks, for the hint. It compiled, but it was rubbish, I think. 😊 

> btw, all that ConsoleWindow stuff is optional and will probably disapear soon

You mean from `monaco-vscode-api`?

Reply:
Yes!